ai/src/Makefile

```diff
@@ -8,7 +8,7 @@ test: venv
 	ALGORITHM="sacd" venv/bin/python3 -m pytest --timeout 30
 
 venv:
-	python3 -m venv venv
+	python3 -m venv -p 3.9 venv
```

Comment:
We should keep using python 3.8 until we upgrade the Dockerfile for the project to use python 3.9 as well. As far as I know, we want to stay on python 3.8 for now as that will allow us to easily support Apple silicon: https://developer.apple.com/metal/tensorflow-plugin/